Issue with osnap when using pnpm

Hello, I recently upgraded to latest version of loki (0.28.1 -> 0.31.1) but I am having some issues when installing and running with pnpm: 
```
> loki

(node:59566) UnhandledPromiseRejectionWarning: Error: Cannot find module '@ferocia-oss/osnap/src/ios'
Require stack:
- loki/examples/react/node_modules/.pnpm/@loki+target-native-ios-simulator@0.31.0/node_modules/@loki/target-native-ios-simulator/src/create-ios-simulator-target.js
...
```
The dependency is missing the `src` folder
![image](https://user-images.githubusercontent.com/1062699/227601367-f747a850-c552-4cc1-b9c8-e87bfc8b0d12.png)

When I ran with yarn, I don't have the same issue. I have the following
![image](https://user-images.githubusercontent.com/1062699/227601898-5ea3af3d-613e-4510-ad24-a702f675843b.png)

I think it is an issue with [osnap](https://github.com/ferocia/osnap) but I could not file an issue on the GitHub project itself.

FYI, I'm using pnpm 7.30.0

Regards,
Pascal.

--- a/packages/target-native-android-emulator/package.json
+++ b/packages/target-native-android-emulator/package.json
@@ -20,7 +20,7 @@
   ],
   "main": "src/index.js",
   "dependencies": {
-    "@ferocia-oss/osnap": "^1.3.0",
+    "@ferocia-oss/osnap": "^1.3.5",
     "@loki/core": "^0.31.0",
     "@loki/target-native-core": "^0.31.0",
     "fs-extra": "^9.1.0",

--- a/packages/target-native-ios-simulator/package.json
+++ b/packages/target-native-ios-simulator/package.json
@@ -20,7 +20,7 @@
   ],
   "main": "src/index.js",
   "dependencies": {
-    "@ferocia-oss/osnap": "^1.3.0",
+    "@ferocia-oss/osnap": "^1.3.5",
     "@loki/core": "^0.31.0",
     "@loki/target-native-core": "^0.31.0",
     "fs-extra": "^9.1.0",

